### ISSUE TITLE
Crypt remote not working

### ISSUE THREAD

**Issue by oxenit** [Magisk-Modules-Repo/com.piyushgarg.rclone#32]
Hi,

I have a Gdrive remote (let's call it drive: ) mounting correctly on my android. 
However the crypt remote (drive_crypt: ) referencing this drive: is not mounting. 

When I type on a terminal`rclone ls drive_crypt:/` I can see all my folders.

Is crypt mounting really supported ?

**Comment by SJ50** [comment]
Yes it works, can you try it without any .global.param or any param file.
